docs/shoot_overlay_network.md

```diff
@@ -0,0 +1,84 @@
+# Enable / disable overlay network for shoots with Calico
+Gardener can be used with or without the overlay network. 
+
+Starting versions:
+- [provider-gcp@v1.25.0](https://github.com/gardener/gardener-extension-provider-gcp/releases/tag/v1.25.0)
+- [provider-alicloud@v1.40.0](https://github.com/gardener/gardener-extension-provider-alicloud/commit/3f3942635c850b443309697c51f66a605fbddc4b)
+- [provider-aws@v1.38.2](https://github.com/gardener/gardener-extension-provider-aws/releases/tag/v1.38.2)
```

Comment:
@DockToFuture / @kon-angelo / @shaoyongfeng: Are those versions still correct? We did some rework after all.

Reply:
About provider-alicloud, the change about overlay network was reverted by https://github.com/gardener/gardener-extension-provider-alicloud/pull/538 for v1.40.0.
this feature will be released again in coming version v1.43.0